LTS.md

```diff
@@ -0,0 +1,25 @@
+# Supported versions
+
+## Long Term Support
+
+Express Long Term Support (LTS) version is provided as per defined in this document - and also include all the packages part of the Express.js Organization (express, pillarjs and jshttp).
+
+1. Express follows a semantic versioning A.B.C (also know as semver), where the major version is represented by "A". Major version are supported for a minimal time of a year (as seen as 365 days). 
```

Comment:
And this is to say every major version is supported a minimum of 1 year? SGTM. My first reading of this doc was a bit confusing because it made it sound like we'll want to do releases every year, the language in Fastify is a bit clearer and helped me understand the intent of this doc.